mono/mini/mini-wasm-debugger.c

```diff
@@ -706,6 +707,32 @@ typedef struct {
 	int *pos;
 } FrameDescData;
 
+/*
+ * this returns a string formatted like
+ *
+ *  <ret_type>:[<comma separated list of arg types>]:<method name>
+ *
+ *  .. which is consumed by `mono_wasm_add_func_var`. It is used for
+ *  generating this for the delegate, and it's target.
+ */
+static GString*
+mono_method_to_desc_for_js (MonoMethod *method, gboolean include_namespace)
```

Comment:
This looks like it could be simplified and reduce allocations with a g_string_printf?  Maybe just moved inline?

Reply:
👍 Fixed.